paddle/phi/api/yaml/fused_ops.yaml

```diff
@@ -358,6 +358,18 @@
     func : layer_norm_act_xpu
     data_type : x
 
+- op : max_pool2d_v2
```

Comment:
Could add some comments to explain the diff between `max_pool2d` and this op?

Reply:
Done